### DISCUSSION TITLE
Filters inside a form or not

### DISCUSSION THREAD

**Discussion by andredewaard** [shadcn-ui/ui#5385] [Q&A]
We are implementing a quite big table with this package that uses @tanstack/table but we will filter and sort the data in the backend.
Shadcn provides an example of how to filter, Why don't we wrap the filters inside a form? 
Is it because in the example we do everything client side and when we will do this server side its better to wrap it in a form?

**Comment by anishshobithps** [comment]
@andredewaard yes, I think there is a example by other user as in like https://github.com/sadmann7/shadcn-table might give you some insights for server side.
